charts/argo-rollouts/values.yaml

```diff
@@ -94,3 +94,29 @@ podLabels: {}
 # Secrets with credentials to pull images from a private registry
 imagePullSecrets: []
 # - name: argo-pull-secret
+
+dashboard:
+  enabled: false
+  component: rollouts-dashboard
+  ## Node selectors and tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  image:
+    registry: quay.io
+    repository: argoproj/kubectl-argo-rollouts
+    tag: ""
+    pullPolicy: IfNotPresent
+  resources: {}
+  podSecurityContext: {}
+  #  Enable runAsNonRoot after image version v1.0.1
+  #  runAsNonRoot: true
+  containerSecurityContext: {}
+  serviceAccount:
+    create: true
+    annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
```

Comment:
Could you also place documentation for these values into the README?

Reply:
Done

Reply:
Perfect. Thank you so much! 🙏